.travis.yml

```diff
@@ -1,14 +1,25 @@
 language: php
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
+  - 5.5.9
   - 5.6
   - hhvm
+  - 7
+
+env:
+  - SYMFONY_VERSION=3.0.*@dev
+
+matrix:
+  allow_failures:
+    - php: 7
+  include:
+    - php: 5.5.9
+    - php: 5.6
+    - php: 7
```

Comment:
These include block can be removed completly. It's only used to overwrite global settings like for example the env variables.